settings.gradle

```diff
@@ -150,3 +150,4 @@ include 'data-prepper-plugins:buffer-common'
 include 'data-prepper-plugins:dissect-processor'
 include 'data-prepper-plugins:dynamodb-source'
 include 'data-prepper-plugins:decompress-processor'
+include 'data-prepper-plugins:flatten-json-processor'
```

Comment:
Does this only work with JSON? Or does it work on the Event model? If it works for the Event model then please rename the project to `flatten-processor`. If it requires JSON, we should change the plugin name to `flatten_json`.

Reply:
It works on the Event model. Will update the project name.

Reply:
Renamed flatten-json to flatten everywhere.